client/src/components/eventForm.jsx

```diff
@@ -0,0 +1,87 @@
+import React, {Component} from 'react'
+import { Card, CardActions, CardHeader, CardMedia, CardTitle, CardText } from 'material-ui/Card';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import TextField from 'material-ui/TextField';
+import DropDownMenu from 'material-ui/DropDownMenu';
+import MenuItem from 'material-ui/MenuItem';
```

Comment:
All of these are being used as smaller components within the event form. DropDown Menu has not yet been implemented.